open_seq2seq/utils/funcs.py

```diff
@@ -74,16 +72,19 @@ def train(config,
   total_time = 0.0
   bench_start = config.get('bench_start', 10)
 
-  #hooks.append(tf_debug.TensorBoardDebugHook("Trantor:6067"))
+  if debug:
+    hooks.append(tf_debug.TensorBoardDebugHook("localhost:6067"))
```

Comment:
can we change this to port number. E.g. go into debug if --debug_port=port-number. if debug_port isn't present then don't go into debug

Reply:
Changed with last commit